lib/ignored-paths.js

```diff
@@ -146,7 +95,7 @@ function IgnoredPaths(options) {
     }
 
     this.defaultPatterns = DEFAULT_IGNORE_PATTERNS.concat(options.patterns || []);
-    this.baseDir = ".";
+    this.baseDir = options.cwd;
 
     this.ig = {
         custom: new ignore.Ignore({
```

Comment:
Thanks for the review and the tips, @kaelzhang!  I'll try out `3.x` and update this PR.